test_vgpu.sh

```diff
@@ -215,10 +234,17 @@ END_OF_VGPU_CONFIRM
 fi
 
 if [ -n "$result" ]; then
-    echo "VGPU create success"
+    echo $VGPU_CREATE_SUCCESS_FLAG
+    echo "###################Check LVM image upload status#####################"
+    nova image-create --poll $VM_NAME $SNAPSHOT_NAME
```

Comment:
Can we also retrieve the image please?

Reply:
Will add another VM create based on the new image

Reply:
done